examples/rbcd_attack_poisoning.py

```diff
@@ -0,0 +1,197 @@
+import functools
+import os.path as osp
+import sys
+from typing import Optional, Tuple
+
+import matplotlib.pyplot as plt
+import torch
+import torch.nn.functional as F
+from torch import Tensor
+
+try:
+    import higher
+except ImportError:
+    sys.exit(
+        'Install `higher` (e.g. `pip install higher`) for poisoning example')
+
+import torch_geometric.transforms as T
+from torch_geometric.datasets import Planetoid
+from torch_geometric.nn import GCNConv, PRBCDAttack
+from torch_geometric.nn.conv.gcn_conv import gcn_norm
+
+dataset = 'Cora'
+path = osp.join(osp.dirname(osp.realpath(__file__)), '..', 'data', 'Planetoid')
+# IMPORTANT: Edge weights are being ignored later and most adjacency
+# preprocessings should be part of the model (part of backpropagation)
+dataset = Planetoid(path, dataset, transform=T.NormalizeFeatures())
+data = dataset[0]
+
+
+class GCN(torch.nn.Module):
```

Comment:
Can we import this model from the previous example (similar to the PointNet examples).